vassalTTF/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>vassalTTF</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>16</maven.compiler.source>
+        <maven.compiler.target>16</maven.compiler.target>
+    </properties>
+    
+</project>
```

Comment:
Was this file added intentionally? It looks like it doesn't belong in the PR.

Reply:
Thanks for asking...I wasn't sure what I was supposed to do with that and figured you'd let me know...Does Maven generate a new one in each branch? Ignore those in the future?

Reply:
Unless you know better, it's possible I accidentally copied it from a short-lived experiment. I wasn't sure if Maven built it or where it came from.

Reply:
I've never seen Maven generate a pom before, so I can't comment about how it got there.

Reply:
OK, I'll fix it now.